omise/views/templates/hook/1.6/card_payment.tpl

```diff
@@ -77,63 +77,49 @@
   // the JS work correctly when the checkout is in one-page mode. It would appear that
   // dynamically created script blocks do not run in the global context
 
+  window.omiseCardFields = [
+    'name',
+    'number',
+    'expiration_month',
+    'expiration_year',
+    'security_code'
+  ];
+
   window.omiseCheckout = function omiseCheckout() {
     if (typeof Omise === 'undefined') {
       alert('{l s='Unable to process the payment, loading the external card processing library is failed. Please contact the merchant.' mod='omise'}');
```

Comment:
Not sure if this is also in a scope of JS cleanup but wonder if we should change the outer single quote to be double quote instead? 
```js
alert("{l s=' blah ' mod='omise'}");
```

Reply:
or I guess it is a PrestaShop magic...